### PR TITLE
add gsoc-2023-eclipse-gradle-kotlin

### PR DESCRIPTION
**Pull request by jamesward**

### CODE REVIEW

**Comment by zoobestik** [comment]
@koshachy, could you proofread please?

File: src/pages/news/gsoc-2023-eclipse-gradle-kotlin/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+**Accurate Diagnostics for syntax errors**
+
+![Accurate Diagnostics for syntax errors](image2.png)
+
+**Signature help**
+
+![Signature help](image7.png)
+
+**Hovering***
```
Comment (zoobestik) [inline src/pages/news/gsoc-2023-eclipse-gradle-kotlin/index.md:41]:
```suggestion
**Hovering**
```

File: src/pages/news/gsoc-2023-eclipse-gradle-kotlin/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+![Syntax highlighting for KTS scripts](image5.png)
+
+**Content assistance**
+
+![Content assistance](image1.png)
+
+**Go-to-definition**
+
+![Go-to-definition](image4.gif)
```
Comment (zoobestik) [inline src/pages/news/gsoc-2023-eclipse-gradle-kotlin/index.md:31]: There is support only `jpg`, `png`, `webp`, `tiff`, `avif` formats 😞
Is it possible to get better quality image in this place?
Screenshot 2023-12-18 at 12 50 27

**Review by zoobestik — CHANGES_REQUESTED** [review]

**Review by koshachy — APPROVED** [review]
Hey there, 
Sorry for the delay. I've slightly updated the markup of the page.
Hope you don't mind.
Ping me if you have any questions.
Thanks!

**Comment by jamesward** [comment]
Thanks! What are the next steps for getting this published?

**Comment by zoobestik** [comment]
@jamesward рi, I'm still having an issue with the animated GIF. ⚠️
I'll fix it today, and it will be done.

**Comment by jamesward** [comment]
That is odd about the GIF as it should work like any other image.  But I haven't tried running the Gatsby server to check it.  Is that where you are seeing the problem?

**Review by zoobestik — APPROVED** [review]

**Comment by jamesward** [comment]
Thank you! And thanks for working through getting that GIF working!

**Pull request merged**